spanner/transaction.go

```diff
@@ -76,6 +76,10 @@ type txReadOnly struct {
 
 // TransactionOptions provides options for a transaction.
 type TransactionOptions struct {
+	// The transaction tag to use for a read/write transaction.
+	// This tag is automatically included with each statement and the commit
+	// request of a read/write transaction.
+	TransactionTag string
```

Comment:
Just wondering why there was empty TransactionOptions struct before this change.

Reply:
We had a major version bump recently, and added a number of breaking changes already back then that we knew would be needed for (among others) tagging support. Adding a struct is not breaking, but we also added a number of new methods that use this struct as a parameter.